aptos-move/aptos-vm-logging/src/lib.rs

```diff
@@ -72,12 +72,7 @@ pub fn speculative_log(level: Level, context: &AdapterLogSchema, message: String
                 alert!("{:?}", e);
             };
         },
-        None => {
-            alert!(
-                "Speculative state not initialized to log message = {}",
-                message
-            );
-        },
+        None => {},
```

Comment:
Remove this error message here as it doesn't play well with multiple blockSTM instances.